test_query.py::test_all `accounts` have no ETH

### The bug

To reproduce, run the following locally.

```console
brownie test util/test/test_query.py::test_all -s
```

It will return with this error: ([full log](https://github.com/oceanprotocol/df-py/files/10314963/test_main1.txt))
```text
        for acc in accounts:
>           oceanutil.create_ve_lock(OCEAN_lock_amt, t2, acc)

util/test/test_query.py:64: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

amount = 5000000000000000000, unlock_time = 1684368000, from_account = <LocalAccount '0xa27cC6b6236E39528F85A7ff97A57233162011F8'>

    def create_ve_lock(amount: float, unlock_time: int, from_account):
>       OCEANtoken().approve(veOCEAN().address, amount, {"from": from_account})
E       ValueError: sender doesn't have enough funds to send tx. The upfront cost is: 48596 and the sender's account only has: 0
```

### Analysis

If we look earlier in test_all(), we find that it's got some strange behavior wrt accounts.

First, before test_all(), it uses setup_function() as follows. Note how in this case it sets account0 = brownie.network.accounts[0]. 
```python
def setup_function():
    global OCEAN_ADDR

    networkutil.connect(networkutil.DEV_CHAINID)
    global account0, QUERY_ST
    account0 = brownie.network.accounts[0]
    QUERY_ST = max(0, len(brownie.network.chain) - 200)
    oceanutil.recordDevDeployedContracts()
    OCEAN_ADDR = oceanutil.OCEAN_address().lower()
```

Below is a snippet of test_all() up until the failing call to create_ve_lock(). Some notes:
- it uses publisher_account = account0, which does have ETH
- it calls `oceantestutil.fillAccountsWithToken(CO2)` which operates on brownie.network.accounts, but then immediately after sets `accounts = []`
- then a few lines later creates 7 new accounts, and transfers CO2 and OCEAN to each of them. It never transfers ETH
```python
def test_all():
...
    CO2_sym = f"CO2_{random.randint(0,99999):05d}"
    CO2 = B.Simpletoken.deploy(CO2_sym, CO2_sym, 18, 1e26, {"from": account0})
    CO2_addr = CO2.address.lower()
    OCEAN = oceanutil.OCEANtoken()
    oceantestutil.fillAccountsWithToken(CO2)
    accounts = []
    publisher_account = account0
    OCEAN_lock_amt = toBase18(5.0)
    for i in range(7):
        accounts.append(brownie.network.accounts.add())
        CO2.transfer(accounts[i], toBase18(11000.0), {"from": account0})
        OCEAN.transfer(accounts[i], OCEAN_lock_amt, {"from": account0})
    sampling_test_accounts = [accounts.pop(), accounts.pop()]

    # Create data nfts
    data_nfts = []
    for i in range(5):
        (data_NFT, DT, exchangeId) = oceanutil.createDataNFTWithFRE(
            publisher_account, CO2
        )
        assert oceanutil.FixedPrice().isActive(exchangeId) is True
        data_nfts.append((data_NFT, DT, exchangeId))

    # Lock veOCEAN
    t0 = brownie.network.chain.time()
    t1 = t0 // WEEK * WEEK + WEEK
    t2 = t1 + WEEK * 20  # lock for 20 weeks
    brownie.network.chain.sleep(t1 - t0)
    for acc in accounts:
        oceanutil.create_ve_lock(OCEAN_lock_amt, t2, acc)
```

--- a/util/test/test_query.py
+++ b/util/test/test_query.py
@@ -36,9 +36,9 @@ def test_all():
     CO2 = B.Simpletoken.deploy(CO2_sym, CO2_sym, 18, 1e26, {"from": account0})
     CO2_addr = CO2.address.lower()
     OCEAN = oceanutil.OCEANtoken()
-    
+
     OCEAN_lock_amt = toBase18(5.0)
-    
+
     accounts = []
     for i in range(7):
         acc = brownie.network.accounts.add()
@@ -46,15 +46,13 @@ def test_all():
         CO2.transfer(acc, toBase18(11000.0), {"from": account0})
         OCEAN.transfer(acc, OCEAN_lock_amt, {"from": account0})
         accounts.append(acc)
-        
+
     sampling_test_accounts = [accounts.pop(), accounts.pop()]
 
     # Create data nfts
     data_nfts = []
     for i in range(5):
-        (data_NFT, DT, exchangeId) = oceanutil.createDataNFTWithFRE(
-            account0, CO2
-        )
+        (data_NFT, DT, exchangeId) = oceanutil.createDataNFTWithFRE(account0, CO2)
         assert oceanutil.FixedPrice().isActive(exchangeId) is True
         data_nfts.append((data_NFT, DT, exchangeId))
 

--- a/util/test/test_query.py
+++ b/util/test/test_query.py
@@ -16,7 +16,6 @@ from util.retry import retryFunction
 account0, QUERY_ST = None, 0
 
 CHAINID = networkutil.DEV_CHAINID
-OCEAN_ADDR: str = ""
 WEEK = 7 * 86400
 
 
@@ -37,21 +36,24 @@ def test_all():
     CO2 = B.Simpletoken.deploy(CO2_sym, CO2_sym, 18, 1e26, {"from": account0})
     CO2_addr = CO2.address.lower()
     OCEAN = oceanutil.OCEANtoken()
-    oceantestutil.fillAccountsWithToken(CO2)
-    accounts = []
-    publisher_account = account0
+    
     OCEAN_lock_amt = toBase18(5.0)
+    
+    accounts = []
     for i in range(7):
-        accounts.append(brownie.network.accounts.add())
-        CO2.transfer(accounts[i], toBase18(11000.0), {"from": account0})
-        OCEAN.transfer(accounts[i], OCEAN_lock_amt, {"from": account0})
+        acc = brownie.network.accounts.add()
+        account0.transfer(acc, toBase18(0.1))
+        CO2.transfer(acc, toBase18(11000.0), {"from": account0})
+        OCEAN.transfer(acc, OCEAN_lock_amt, {"from": account0})
+        accounts.append(acc)
+        
     sampling_test_accounts = [accounts.pop(), accounts.pop()]
 
     # Create data nfts
     data_nfts = []
     for i in range(5):
         (data_NFT, DT, exchangeId) = oceanutil.createDataNFTWithFRE(
-            publisher_account, CO2
+            account0, CO2
         )
         assert oceanutil.FixedPrice().isActive(exchangeId) is True
         data_nfts.append((data_NFT, DT, exchangeId))
@@ -82,7 +84,7 @@ def test_all():
     # pylint: disable=consider-using-enumerate
     for i in range(len(accounts)):
         oceantestutil.buyDTFRE(data_nfts[i][2], 1.0, 10000.0, accounts[i], CO2)
-        oceantestutil.consumeDT(data_nfts[i][1], publisher_account, accounts[i])
+        oceantestutil.consumeDT(data_nfts[i][1], account0, accounts[i])
 
     # sampling test accounts locks and allocates after start block
     # pylint: disable=consider-using-enumerate
@@ -585,14 +587,11 @@ def test_retryFunction_query():
 
 @enforce_types
 def setup_function():
-    global OCEAN_ADDR
-
     networkutil.connect(networkutil.DEV_CHAINID)
     global account0, QUERY_ST
     account0 = brownie.network.accounts[0]
     QUERY_ST = max(0, len(brownie.network.chain) - 200)
     oceanutil.recordDevDeployedContracts()
-    OCEAN_ADDR = oceanutil.OCEAN_address().lower()
 
 
 @enforce_types

